### PR TITLE
chore: [Hub Bot] Refresh metadata 2023-08-30

### PR DESCRIPTION
**Pull request by MeltyBot**
Updates Plugin Definitions

@tayloramurphy @sbalnojan as part of https://github.com/meltano/hub/issues/1280 I made some fixes to more consistently scrape metadata and I opened it up to make changes to all SDK plugins listed, previously it was only looking at meltanolabs variants, this PR is the result.

I reviewed this PR pretty well already and noted any weird edge cases that I need to fix but it should be in good shape now. This PR makes a lot of minor consistency changes like alpha sorting settings and adding default SDK settings like stream map and batch. After this PR and all the one time minor changes we should see a much smaller diff on a weekly basis, with actual changes to the taps.

### CODE REVIEW

File: _data/meltano/extractors/tap-cassandra/datarts-tech.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ settings:
   kind: password
   label: Password
   name: password
-- description: The server-side port to open connections to. Defaults to 9042..
+- description: The server-side port to open connections to. Defaults to 9042. .
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-cassandra/datarts-tech.yml:79]:
```suggestion
- description: The server-side port to open connections to. Defaults to 9042..
```

File: _data/meltano/extractors/tap-cassandra/datarts-tech.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ settings:
   name: keyspace
 - description: The local_dc parameter should be the name of the datacenter.
   kind: string
-  label: Local Datacenter
+  label: Local Dc
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-cassandra/datarts-tech.yml:68]:
```suggestion
  label: Local Datacenter
```

File: _data/meltano/extractors/tap-freshbooks/hotgluexyz.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@ settings:
   name: flattening_max_depth
 - description: The API redirect URI.
   kind: string
-  label: Redirect URI
+  label: Redirect Uri
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-freshbooks/hotgluexyz.yml:42]:
```suggestion
  label: Redirect URII
```

File: _data/meltano/extractors/tap-gmail/kgpayne.yml [hunk 4]
```diff
@@ -20,44 +20,53 @@ pip_url: meltanolabs-tap-gmail
...
+  label: Flattening Max Depth
+  name: flattening_max_depth
+- description: Include messages from SPAM and TRASH in the results.
+  kind: boolean
+  label: Messages Include Spam Trash
+  name: messages.include_spam_trash
+  value: false
+- description: Only return messages matching the specified query. Supports the same
+    query format as the Gmail search box. For example, "from:someuser@example.com
+    rfc822msgid:<somemsgid@example. Com> is:unread". Parameter cannot be used when
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-gmail/kgpayne.yml:39]:
```suggestion
    rfc822msgid:<somemsgid@example.com> is:unread". Parameter cannot be used when
```

File: _data/meltano/extractors/tap-gmail/meltanolabs.yml [hunk 5]
```diff
@@ -19,44 +19,53 @@ pip_url: meltanolabs-tap-gmail
...
+  label: Flattening Max Depth
+  name: flattening_max_depth
+- description: Include messages from SPAM and TRASH in the results.
+  kind: boolean
+  label: Messages Include Spam Trash
+  name: messages.include_spam_trash
+  value: false
+- description: Only return messages matching the specified query. Supports the same
+    query format as the Gmail search box. For example, "from:someuser@example.com
+    rfc822msgid:<somemsgid@example. Com> is:unread". Parameter cannot be used when
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-gmail/meltanolabs.yml:38]:
```suggestion
    rfc822msgid:<somemsgid@example.com> is:unread". Parameter cannot be used when
```

File: _data/meltano/extractors/tap-google-analytics/sehnem.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+pip_url: git+https://github.com/sehnem/tap-google-analytics.git
+quality: silver
+repo: https://github.com/sehnem/tap-google-analytics
+settings:
+- description: Google Analytics Access Token
+  kind: password
+  label: Access Token
+  name: access_token
+- description: Google Analytics Client ID
+  kind: password
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-google-analytics/sehnem.yml:29]: Delete this

File: _data/meltano/extractors/tap-google-analytics/connorflyn.yml [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+  kind: password
+  label: OAuth Credentials Client Secret
+  name: oauth_credentials.client_secret
+- description: Google Analytics Refresh Token
+  kind: password
+  label: OAuth Credentials Refresh Token
+  name: oauth_credentials.refresh_token
+- description: Google Analytics Property ID
+  kind: password
+  label: Property ID
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-google-analytics/connorflyn.yml:63]: Delete this

File: _data/meltano/extractors/tap-gorgias/brooklyn-data.yml [hunk 8]
```diff
@@ -18,26 +19,47 @@ pip_url: git+https://github.com/brooklyn-data/tap-gorgias.git
...
+- description: User-defined config values to be used within map expressions.
+  kind: object
+  label: Stream Map Config
+  name: stream_map_config
+- description: Config object for stream maps capability. For more information check
+    out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html).
+  kind: object
+  label: Stream Maps
+  name: stream_maps
+- description: Subdomain (<subdomain>. Gorgias. Com)
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-gorgias/brooklyn-data.yml:57]:
```suggestion
- description: Subdomain (<subdomain>.gorgias.com)
```

File: _data/meltano/extractors/tap-hubspot/spacecowboy.yml [hunk 9]
```diff
@@ -17,15 +20,62 @@ pip_url: git+https://github.com/spacecowboy/tap-hubspot.git
...
+  label: Flattening Enabled
+  name: flattening_enabled
+- description: The max depth to flatten schemas.
+  kind: integer
+  label: Flattening Max Depth
+  name: flattening_max_depth
+- description: HubSpot private app token
   kind: password
-  label: HubSpot API Key
+  label: Hapikey
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-hubspot/spacecowboy.yml:56]:
```suggestion
  label: HubSpot API Key
```

File: _data/meltano/extractors/tap-jira/meltanolabs.yml [hunk 10]
```diff
@@ -21,27 +21,10 @@ pip_url: git+https://github.com/MeltanoLabs/tap-jira.git
 quality: silver
 repo: https://github.com/MeltanoLabs/tap-jira
 settings:
-- description: The authentication type to use.
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-jira/meltanolabs.yml:24]: This didnt parse properly

File: _data/meltano/extractors/tap-mailchimp/acarter24.yml [hunk 11]
```diff
@@ -27,10 +27,10 @@ settings:
   name: api_key
 - description: Your Mailchimp DC
   kind: string
-  label: Data Center
+  label: Dc
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-mailchimp/acarter24.yml:30]:
```suggestion
  label: Data Center
```

File: _data/meltano/extractors/tap-meltano/matatika.yml [hunk 12]
```diff
@@ -15,8 +18,10 @@ pip_url: git+https://github.com/Matatika/tap-meltano.git
 quality: gold
 repo: https://github.com/Matatika/tap-meltano
 settings:
-- description: The target Meltano project's database URI
-  kind: password
-  label: Meltano Database URI
+- description: Meltano Database URI you want to get jobs from
+  kind: string
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-meltano/matatika.yml:22]:
```suggestion
  kind: password
```

File: _data/meltano/extractors/tap-netsuite/sehnem.yml [hunk 13]
```diff
@@ -16,50 +20,82 @@ pip_url: git+https://github.com/sehnem/tap-netsuite.git
...
+  label: Cache Wsdl
+  name: cache_wsdl
+  value: true
+- description: The netsuite account code consumer key
   kind: password
-  label: NetSuite Consumer Key
+  label: Consumer Key
   name: consumer_key
-- description: Your consumer secret for token based authentication consumer key for
-    SOAP connection. Visit the https://support.cazoomi.com/hc/en-us/articles/360010093392-How-to-Setup-NetSuite-Token-Based-Authentication-as-Authentication-Type
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-netsuite/sehnem.yml:30]: Push these docs upstream.

File: _data/meltano/extractors/tap-pinterest-ads/gthesheep.yml [hunk 14]
```diff
@@ -15,20 +17,7 @@ namespace: tap_pinterest_ads
 pip_url: git+https://github.com/gthesheep/tap-pinterest-ads.git
 quality: silver
 repo: https://github.com/gthesheep/tap-pinterest-ads
-settings:
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-pinterest-ads/gthesheep.yml:18]: Not configured in the tap properly.

File: _data/meltano/extractors/tap-rest-api-msdk/widen.yml [hunk 15]
```diff
@@ -241,7 +241,7 @@ settings:
     tap will dynamically update the token with either the `start_date`or the last
     bookmark / state value. A simple template Example for FHIR API's: gt$last_run_date.
     A more complex example against an Opensearch API, {\"bool\": {\"filter\": [{\"\
-    range\": { \"meta.lastUpdated\": { \"gt\": \"$last_run_date\" }}}] }}. Note: Any
+    range\": { \"meta.lastUpdated\": { \"gt\": \"$last_run_date\" }}}] }} .Note: Any
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-rest-api-msdk/widen.yml:244]:
```suggestion
    range\": { \"meta.lastUpdated\": { \"gt\": \"$last_run_date\" }}}] }}. Note: Any
```

File: _data/meltano/extractors/tap-saasoptics/datarts-tech.yml [hunk 16]
```diff
@@ -85,7 +85,7 @@ settings:
   kind: object
   label: Stream Maps
   name: stream_maps
-- description: tap-saasoptics <api_user_email@your_company.com>.
+- description: Tap-saasoptics <api_user_email@your_company. Com>.
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-saasoptics/datarts-tech.yml:88]:
```suggestion
- description: tap-saasoptics <api_user_email@your_company.com>.
```

File: _data/meltano/extractors/tap-salesforce/hotgluexyz.yml [hunk 17]
```diff
@@ -0,0 +1,64 @@
+capabilities:
+- about
+- catalog
+- discover
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-salesforce/hotgluexyz.yml:4]: Remove

File: _data/meltano/extractors/tap-totango/edsoncezar16.yml [hunk 18]
```diff
@@ -66,21 +68,24 @@ settings:
...
+  value: []
+- description: The url for the API services. Https://api. Totango. Com is for US services,
     whereas https://api-eu1.totango.com is for EU services.
   kind: options
   label: API URL
   name: api_url
   options:
-  - value: https://api.totango.com
-  - value: https://api-eu1.totango.com
+  - label: Https://API Totango Com
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-totango/edsoncezar16.yml:78]:
```suggestion
  - label: https://api.totango.com
```

File: _data/meltano/extractors/tap-totango/edsoncezar16.yml [hunk 19]
```diff
@@ -66,21 +68,24 @@ settings:
...
+    value: https://api.totango.com
+  - label: 'Https://API Eu1 Totango Com '
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-totango/edsoncezar16.yml:80]:
```suggestion
  - label: 'https://api-eu1.totango.com '
```

File: _data/meltano/extractors/tap-tradera/kgpayne.yml [hunk 20]
```diff
@@ -19,39 +19,47 @@ pip_url: git+https://github.com/kgpayne/tap-tradera.git
...
 repo: https://github.com/kgpayne/tap-tradera
 settings:
-- description: The Tradera.com developer app ID
+- description: The Tradera. Com developer app ID
   kind: password
-  label: App Id
+  label: App ID
   name: app_id
-- description: The Tradera.com developer app key
+- description: The Tradera. Com developer app key
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-tradera/kgpayne.yml:26]:
```suggestion
- description: The Tradera.com developer app key
```

File: _data/meltano/extractors/tap-trello/matatika.yml [hunk 21]
```diff
@@ -16,17 +19,34 @@ pip_url: git+https://github.com/Matatika/tap-trello.git
 quality: gold
 repo: https://github.com/Matatika/tap-trello
 settings:
-- description: Your Trello developer API key. [How to get your developer API key](https://github.com/Matatika/tap-trello#source-authentication-and-authorization)
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-trello/matatika.yml:19]: Add to tap

File: _data/meltano/extractors/tap-vilkas/hotgluexyz.yml [hunk 22]
```diff
@@ -20,12 +20,8 @@ pip_url: git+https://gitlab.com/hotglue/tap-vilkas.git
 quality: gold
 repo: https://gitlab.com/hotglue/tap-vilkas
 settings:
-- description: The API base URL.
-  kind: string
-  label: Base URL
-  name: base_url
-- description: "'True' to enable schema flattening and automatically expand nested\
-    \ properties."
+- description: "'True' to enable schema flattening and automatically expand nested
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-vilkas/hotgluexyz.yml:23]:
```suggestion
- description: The API base URL.
  kind: string
  label: Base URL
  name: base_url
- description: "'True' to enable schema flattening and automatically expand nested
```

File: _data/meltano/loaders/target-mongodb/hotgluexyz.yml [hunk 23]
```diff
@@ -1,20 +1,22 @@
...
 - about
+- stream-maps
 description: General purpose, document-based, distributed database.
 domain_url: https://www.mongodb.com/
 keywords:
 - meltano_sdk
 - database
 label: MongoDB
 logo_url: /assets/logos/loaders/mongodb.png
+maintenance_status:
```
Comment (pnadolny13) [inline _data/meltano/loaders/target-mongodb/hotgluexyz.yml:11]:
```suggestion
maintenance_status: active
```

File: _data/meltano/loaders/target-mongodb/hotgluexyz.yml [hunk 24]
```diff
@@ -1,20 +1,22 @@
...
 name: target-mongodb
 namespace: target_mongodb
 pip_url: git+https://gitlab.com/hotglue/target-mongodb.git
 quality: gold
 repo: https://gitlab.com/hotglue/target-mongodb
 settings:
 - description: Connection string in the format mongodb://[${MONGO_USERNAME:${MONGO_PASSWORD}@]mongodb0.example.com:PORT
-  kind: password
+  kind: string
```
Comment (pnadolny13) [inline _data/meltano/loaders/target-mongodb/hotgluexyz.yml:19]:
```suggestion
  kind: password
```

File: _data/meltano/loaders/target-s3/crowemi.yml [hunk 25]
```diff
@@ -29,20 +29,162 @@ settings:
...
-    value: day
-  - label: Hour
-    value: hour
-  - label: Minute
-    value: minute
-  - label: Second
-    value: second
-  - label: Microsecond
-    value: microsecond
+  - label: D
```
Comment (pnadolny13) [inline _data/meltano/loaders/target-s3/crowemi.yml:32]: Revert this

File: _data/meltano/loaders/target-s3/jkausti.yml [hunk 26]
```diff
@@ -0,0 +1,72 @@
+capabilities:
+- about
+- schema-flattening
+- stream-maps
+description: AWS S3 - Apache Parquet File Format
+domain_url: https://aws.amazon.com/s3/
+executable: target-s3
+keywords:
+- meltano_sdk
```
Comment (pnadolny13) [inline _data/meltano/loaders/target-s3/jkausti.yml:9]: delete this

File: _data/meltano/extractors/tap-pinterest-ads/gthesheep.yml [hunk 27]
```diff
@@ -15,20 +17,7 @@ namespace: tap_pinterest_ads
...
-  label: Client Secret
-  name: client_secret
-- description: Refresh token obtained from the OAuth user flow, described [here](https://developers.pinterest.com/docs/api/v5/#tag/Authentication)
-  kind: password
-  label: Refresh Token
-  name: refresh_token
-- description: Start date to collect ad analytics from
-  label: Start Date
-  name: start_date
+settings: []
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-pinterest-ads/gthesheep.yml:20]:
```suggestion
settings:
- description: App ID for your Pinterest App
  label: Client ID
  name: client_id
- description: App secret key
  label: Client Secret
  name: client_secret
- description: Refresh token obtained from the OAuth user flow, described [here](https://developers.pinterest.com/docs/api/v5/#tag/Authentication)
  kind: password
  label: Refresh Token
  name: refresh_token
- description: Start date to collect ad analytics from
  label: Start Date
  name: start_date
```

**Review by tayloramurphy — APPROVED** [review]
Lots of changes. Approving!

**Pull request merged**